server/views/admin/placementRequests/show.njk

```diff
@@ -2,10 +2,16 @@
 {%- from "moj/components/identity-bar/macro.njk" import mojIdentityBar -%}
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 
+{% from "../../components/keyDetails/macro.njk" import keyDetails %}
 {% extends "../../partials/layout.njk" %}
 
 {% set pageTitle = applicationName + " - View Placement Request" %}
 
+{% block header %}
+    {{ super() }}
+    {{ keyDetails(MatchUtils.keyDetails(placementRequest)) }}
```

Comment:
No strictly related to the changes you made, in CAS-3 we have a very similiar component to `keyDetails` called `placeContextHeader` (not a big fan of the name 🤷 ) so wondered `keyDetails` was implemented.  Not sure if its worth doing as part of these changes but I think the markup in `keyDetails` could be improved to be more semantically structured to help screen reader users. Also I'm not sure how well this particular implementation works with mobil but I know before we refactored placeContextHeader it looked terrible on mobil or zoomed in.

This is the PR we did when we refactored the component https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/1026 and here is the component itself https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/server/views/components/place-context-header/macro.njk.  For mobile/zoomed in view we change the layout of the `dl` items so they are column layout instead inline